app/com/arpnetworking/metrics/portal/scheduling/impl/DatabaseExecutionHelper.java

```diff
@@ -73,6 +73,8 @@ public DatabaseExecutionHelper(
      * @return An internal model for this execution
      */
     public static <T, E extends BaseExecution<T>> JobExecution<T> toInternalModel(final E beanModel) {
+        // TODO(cbriones) - This should not be static, repositories should call this from an instance.
```

Comment:
If this is not in your development plan to address with Alerting, please file an issue for it against the OSS project.

Reply:
Filed https://github.com/ArpNetworking/metrics-portal/issues/384